src/main/java/liquibase/ext/databricks/snapshot/jvm/ColumnSnapshotGeneratorDatabricks.java

```diff
@@ -120,4 +121,30 @@ private String sanitizeStatement(String createTableStatement) {
         }
         return sanitizedCreateTableStatement;
     }
+
+    private Column.AutoIncrementInformation parseAutoIncrementInfo(String statement, String columnName) {
+        String findLineRegex = "(\\b"+columnName+"\\s+\\w+(\\s+NOT\\s+NULL)?\\s+GENERATED\\s+(BY\\s+DEFAULT|ALWAYS)\\s+AS\\s+IDENTITY\\s+\\(START WITH (\\d+) INCREMENT" +
```

Comment:
Looks like per documentation `START WITH` and `INCREMENT BY` parts are optional and could be specified both none of them or any of them.  Might be necessary to  group them and expect from 0 to 1 appearences.

Reply:
I'd suggest `(\bcolumnName\s+\w+(\s+NOT\s+NULL)?\s+GENERATED\s+(BY\s+DEFAULT|ALWAYS)\s+AS\s+IDENTITY\s*(\((START WITH (\d+)){0,1}( INCREMENT BY (\d+)){0,1}\)){0,1})`

Reply:
Even when creating table without provding `START WITH 1 INCREMENT BY 1`
```
CREATE TABLE main.oleh.tableWithDefaultValues1 (
  longcolumn BIGINT GENERATED BY DEFAULT AS IDENTITY,
...
```
`show create table` statement add them anyway 
```
CREATE TABLE main.oleh.tableWithDefaultValues1 (
  longcolumn BIGINT GENERATED BY DEFAULT AS IDENTITY (START WITH 1 INCREMENT BY 1),
```